packages/aws-cdk-lib/aws-eks/lib/alb-controller.ts

```diff
@@ -125,7 +130,7 @@ export class AlbControllerVersion {
   /**
    * v2.5.1
    */
-  public static readonly V2_5_1 = new AlbControllerVersion('v2.5.1', '1.5.2', false);
+  public static readonly V2_5_1 = new AlbControllerVersion('v2.5.1', '1.5.1', false);
```

Comment:
Not all chart version would aligned with App version, for the version `v2.5.1` the chart version should be `1.5.2`.

You may check version mapping with the command below
```sh
helm repo update
helm search repo aws-load-balancer-controller --versions
```

Reply:
Thanks for the feedback, seems a lot more practical than using git tags :D I'll integrate that into my tool and update everything in a bit 👍

Reply:
Yup, sometimes there might have minor update for "chart" after "app" version released.

For `v2.5.1`, you may check it with the command below to ensure you are on the right track.
```sh
helm search repo aws-load-balancer-controller --versions --output json | jq '[.[] | select(.app_version == "v2.5.1")]'
```

Output:
```json
[
  {
    "name": "eks/aws-load-balancer-controller",
    "version": "1.5.2",
    "app_version": "v2.5.1",
    "description": "AWS Load Balancer Controller Helm chart for Kubernetes"
  },
  {
    "name": "eks/aws-load-balancer-controller",
    "version": "1.5.1",
    "app_version": "v2.5.1",
    "description": "AWS Load Balancer Controller Helm chart for Kubernetes"
  }
]
```

Reply:
There's more about [helm search repo](https://helm.sh/docs/helm/helm_search_repo/), check it out!

Reply:
Yeah that's the command I'm going with, I already have all the SDK versions fetched so I'll just do the one request in JSON output and find each corresponding helm version, shouldn't take long, I just won't be able to work on it right away. Thanks again 👍

Reply:
Fixed 👍